### PR TITLE
Add The Mind

### PR DESCRIPTION
**Pull request by ruthiel**
Reference to #131
Thank you for the opportunity!

Submitting this PR means you have:

- [X] searched the [readme](https://github.com/edm00se/awesome-board-games/blob/master/readme.md) file and searched the [open pull requests](https://github.com/edm00se/awesome-board-games/pulls) (no duplicates please)
- [X] and will follow [the desired format](https://github.com/edm00se/awesome-board-games/blob/master/formatting.md)
- [X] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines](https://github.com/edm00se/awesome-board-games/blob/master/contributing.md) or [open an issue](https://github.com/edm00se/awesome-board-games/issues/new).

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -802,6 +802,16 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
...
+
+> The Mind is more than just a game. It's an experiment, a journey, a team experience in which you can't exchange information, yet will become one to defeat all the levels of the game. In more detail, the deck contains cards numbered 1-100, and during the game you try to complete 12, 10, or 8 levels of play with 2, 3, or 4 players. In a level, each player receives a hand of cards equal to the number of the level: one card in level 1, two cards in level 2, etc. Collectively you must play these cards into the center of the table on a single discard pile in ascending order but you cannot communicate with one another in any way as to which cards you hold. You simply stare into one another's eyes, and when you feel the time is right, you play your lowest card. If no one holds a card lower than what you played, great, the game continues! If someone did, all players discard face up all cards lower than what you played, and you lose one life.
+
+![The Mind image](https://cf.geekdo-images.com/q_JGK291hrhnhiRB0667oA__imagepage/img/WUf5FT5SWkXQsf-K1Q3-fHyyJYY=/fit-in/900x600/filters:no_upscale():strip_icc()/pic3979766.png)
```
Comment (edm00se) [inline readme.md:809]:
```suggestion
![The Mind image](https://cf.geekdo-images.com/itemrep/img/w6x_ahGLQVe2u6trDx0kO91OprM=/fit-in/246x300/filters:strip_icc()/pic3979766.png)
```

The image is a bit larger than the recommended format size. This tile image would suit it better.

**Review by edm00se — CHANGES_REQUESTED** [review]
Please update the image, an alternative that meets the targeted image size has been suggested. Otherwise this PR looks good.

**Review by edm00se — APPROVED** [review]
Thanks for contributing. Happy Hacktoberfest!

**Comment by ruthiel** [comment]
Thank you for your initiative!
Happy Hacktoberfest! =)

**Pull request merged**